src/core/public/core_system.ts

```diff
@@ -110,6 +111,7 @@ export class CoreSystem {
 
   private readonly rootDomElement: HTMLElement;
   private readonly coreContext: CoreContext;
+  private readonly workspaces: WorkspaceService;
```

Comment:
Do we need to rename variable to `workspace` or rename service name to `WorkspacesService`? Just find they are using consistent name in above.